Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/handler/ChangePlayerStatusMessageHandler.java

```diff
@@ -27,9 +29,6 @@ protected void handle(Connection connection, ChangePlayerStatusMessage message)
         }
 
         ChangePlayerStatusMessage.Status status = message.getStatus();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS106)